### PR TITLE
add new screen for which sector other

### PR DESCRIPTION
**Pull request by swalke-ch**
add new screen for which sector other

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/config/index.ts [hunk 1]
```diff
@@ -37,3 +37,4 @@ export const NAME_REGISTERED_WITH_AML = `${BASE_COMMON_URL}/name-registered-with
 export const WHAT_IS_YOUR_ROLE = `${BASE_COMMON_URL}/what-is-your-role/what-is-your-role`;
 export const STOP_NOT_RELEVANT_OFFICER_KICK_OUT = `${BASE_COMMON_URL}/stop-not-relevant-officer/stop-not-relevant-officer`;
 export const SECTOR_YOU_WORK_IN = `${BASE_COMMON_URL}/sector-you-work-in/sector-you-work-in`;
+export const WHICH_SECTOR_YOTHER = `${BASE_COMMON_URL}/which-sector-other/which-sector-other`;
```
Comment (sampankumar) [inline src/main/config/index.ts:40]: Is there a typo here in the constant name?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/routes/index.ts [hunk 2]
```diff
@@ -41,6 +42,9 @@ routes.post(urls.SOLE_TRADER_WHAT_IS_YOUR_NAME, nameValidator, soleTraderNameCon
 routes.get(urls.SOLE_TRADER_SECTOR_YOU_WORK_IN, soleTraderSectorYouWorkInController.get);
 routes.post(urls.SOLE_TRADER_SECTOR_YOU_WORK_IN, sectorYouWorkInValidator, soleTraderSectorYouWorkInController.post);
 
+routes.get(urls.SOLE_TRADER_WHICH_SECTOR_OTHER, whichsectorOtherController.get);
+routes.post(urls.SOLE_TRADER_WHICH_SECTOR_OTHER, whichSectorOtherValidator, whichsectorOtherController.post);
```
Comment (sardhani-ch) [inline src/main/routes/index.ts:46]: If you plan to use same controller for all three journeys, can you move the controller to controllers/features/common.
if not rename controller to soleTraderWhichSectorOtherController to differentiate between three controllers

File: src/main/config/index.ts (hunk 1 above)
Comment (swalke-ch) [reply src/main/config/index.ts]: fixed it now

**Review by sampankumar — DISMISSED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sardhani-ch — APPROVED** [review]

**Review by sampankumar — DISMISSED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by hemerem-ch — APPROVED** [review]

**Pull request merged**